Portfolio/index.css

```diff
@@ -0,0 +1,772 @@
+/* navigation Bar */
+
+*{
+    margin: 0px;
+    padding: 0px;
+    border: 0px;
+
+}
+body{
+    background-color:#004572;
+}
+.navbar {
+    position: relative;
+    
+    cursor: url('/cursor.png'),auto;
+ /* position: relative; */
+ 
+ /* border: 2px solid red; */
+ width: 96.3%;
+    display: inline-flex;
+    align-items: center;
+    justify-content: space-between; 
+    padding: 10px 23px;
+    background-color:#003E66;
+    color: #00D8FF;
+    position: relative;
+}
+
+.navbar h1 {
+    font-family: Revalia;
+    margin: 0;
+    font-size: 2rem;
+}
+
+.navbar ul {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+    display: flex;
+    padding: 1px 215px;

```

Comment:
The padding values seem contradictory, consider reviewing and adjusting as needed.